gt/lua_lpm.c

```diff
@@ -196,6 +196,57 @@ l_lpm_lookup(lua_State *l)
 	return 1;
 }
 
+/*
+ * Converts a given depth value to its corresponding mask value.
+ */
+static uint32_t
+depth_to_mask(uint8_t depth)
+{
+	if ((depth == 0) || (depth > RTE_LPM_MAX_DEPTH)) {
+		rte_panic("LPM: Invalid depth (%u) at line %s",
+			(unsigned)(depth), __func__);
+	}
```

Comment:
This block of code is doing the same `RTE_VERIFY()` is meant to do. Use `RTE_VERIFY()`.

The same comment applies to the IPv6 version.